src/BudgetManager/BudgetManager.CQRS/Responses/TransactionResponses/TransactionResponse.cs

```diff
@@ -1,27 +1,15 @@
-﻿using BudgetManager.Model.Enums;
-
-namespace BudgetManager.CQRS.Responses.TransactionResponses
+﻿namespace BudgetManager.CQRS.Responses.TransactionResponses
 {
     public class TransactionResponse
     {
```

Comment:
Category type should also be included.

Reply:
There's actually a property called CategoryName, which is the string representation of OperationType enum.

Reply:
CategoryType is responsible for enum, CategoryName is for example "Shopping".